devito/__init__.py

```diff
@@ -1,4 +1,5 @@
 from itertools import product
+from . import _version
```

Comment:
All below is autogenerated by versioneer, those files were outdated and not compatible with python3.12